src/main/java/javaClasses/homework_4/stepan_alvajyan/Work.java

```diff
@@ -0,0 +1,18 @@
+package javaClasses.homework_4.stepan_alvajyan;
+
+public class Work {
+    public static void main(String[] args) {
+        Developer dev = new Developer();
```

Comment:
Please overload constructor for Developer and QualityAssurance classes, so they can accept arguments, to initialize your object without using setters (Developer("Name","LastName","techStack"))